pkg/diagnostics/workflow_metrics_test.go

```diff
@@ -0,0 +1,326 @@
+package diagnostics
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"go.opencensus.io/stats/view"
+	"testing"
+)
+
+const (
+	workflowName = componentName
```

Comment:
Why use this const `WorkflowName` in the tests. You can simply use `componentName`. Also the name `workflowName` would be incorrect as it's actually the component name only, not the workflow name.

Reply:
Sure, changing that.